docs/src/runes.md

```diff
@@ -0,0 +1,158 @@
+a transaction's inputs to its outputs.
+
+A transaction output may hold balances of any number of runes.
+
+Runes are identified by IDs, which consist of the block in which a rune was
+etched and the index of the etching transaction within that block.
+
+Etching
+-------
+
+Runes come into existence by being etched. Etching creates a rune and sets its
+properties. Once set, these properties are immutable, even to its etcher.
+
+### Name
+
+Names consist of the letters A through Z and are between one and twenty-eight
+characters long. For example `UNCOMMONGOODS` is a rune name.
+
+Names may contain spacers, represented as bullets, to aid readability.
+`UNCOMMONGOODS` might be etched as `UNCOMMON•GOODS`.
+
+The uniqueness of a name does not depend on spacers. Thus, a rune may not be
+etched with the same sequence of letters as an existing rune, even if it has
+different spacers.
+
+### Divisibility
+
+A rune's divisibility is how finely it may be divided into its atomic units.
+Divisibility is expressed as the number of digits permissible after the decimal
+point in an amount of runes. A rune with divisibility 0 may not be divided. A
+unit of a rune with divisibility 1 may be divided into ten sub-units, a rune
+with divisibility of 2 may be divided into a hundred, and so on.
+
+### Symbol
+
+A rune's currency symbol is a single Unicode code point, for example `$`, `⧉`,
+or `🧿`, displayed after quantities of that rune.
+
+101 atomic units of a rune with divisibility 2 and symbol `🧿` would be
+rendered as `1.01 🧿`.
+
+If a rune does not have a symbol, the generic currency sign `¤`, also called a
+scarab, should be used.
+
+### Premine
+
+The etcher of a rune may optionally allocate to themselves units of the rune
+being etched. This allocation is called a premine.
+
+### Terms
+
+A rune may have an open mint, allowing anyone to create and allocate units of
+that rune for themselves. An open mint is subject to terms, which are set upon
+etching.
+
+A mint is open while all terms of the mint are satisfied, and closed when any
+of them are not. For example, a mint may be limited to a starting height, an
+ending height, and a cap, and will be open between the starting height and
+ending height, or until the cap is reached, whichever comes first.
+
+#### Cap
+
+The number of times a rune may be minted is its cap. A mint is closed once the
+cap is reached.
+
+#### Amount
+
+Each mint transaction creates a fixed amount of new units of a rune.
+
+#### Start Height
+
+A mint is open starting in the block with the given start height.
+
+#### End Height
+
+A rune may not be minted in or after the block with the given end height.
+
+#### Start Offset
+
+A mint is open starting in the block whose height is equal to the start offset
+plus the height of the block in which the rune was etched.
+
+#### End Offset
+
+A rune may not be minted in or after the block whose height is equal to the end
+offset plus the height of the block in which the rune was etched.
+
```

Comment:
What is the behavior if neither a premine nor terms are specified? It seems the default for each of those is 0/do nothing. Does that etch a valid rune with 0 max supply (effectively "burning" the name), or does it create a cenotaph (and burn all runes that are also on that tx)?

Reply:
If neither premine nor terms are supplied, the supply is zero, but the runestone is not a cenotaph.